text/0000-bots-for-health.md

```diff
@@ -0,0 +1,93 @@
+*   Start date: 2020-11-04
+*   Scope: Collective
+*   RFC PR: <!-- leave this empty -->
+*   Implementation issue(s): <!-- leave this empty -->
+
+# Summary
+
+It’s becoming harder to handle the incoming stream of questions and issues, so I
+thought I’d open a discussion to talk about how to tackle that.
+
+Bots may help.
+
+This is a conversation starter: **I’m curious to hear what y’all think would
+(not) be useful**.
+
+## Motivation
+
+It’s becoming harder to handle the incoming stream of questions and issues, so I
+thought I’d open a discussion to talk about how to tackle that.
+
+I’m assuming this is coming from a couple of places:
+
+*   unified is becoming more popular
+*   MDX, recently react-markdown, and potentially more in the future, joined
+*   [Discussions](https://github.blog/2020-05-06-new-from-satellite-2020-github-codespaces-github-discussions-securing-code-in-private-repositories-and-more/#discussions) is an easy way for folks to ask questions
+
+Regardless of why, a different type of user is reaching unified (an exemplary
+question yesterday was: “why are there margins on my `<p>`”,
+[remarkjs/react-markdown#503](https://github.com/remarkjs/react-markdown/issues/503)).
+A lot of these questions are XY problems and result in maintainers acting like
+a rubber duck for three back-and-forths.
+
+I hope that automation can help deal with the flow of incoming questions so that
+maintainers can stay healthy.
+
+## Detailed design
+
+For a solution, here are a couple broad ideas:
+
+*   Improve docs on unifiedjs.com (important, but to be discussed somewhere
+    else)
+*   Improve contributing, support, and issue templates
+*   Shared saved responses (see
+    <https://github.com/electron/governance/tree/master/playbooks/responses>
+    and
+    <https://github.com/electron/electron/issues?q=is%3Aissue+is%3Aclosed>)
+*   Add automation/bots
+
+Specifically, for automation, I’m thinking:
+
+*   Enforce filled out issue/pr/discussion templates
+*   “Lint” issue checklists: if someone is on an old Node, respond and close
+    that those aren’t supported; or enforce that
+    `* [ ] I promise to adhere to the code of conduct` is checked
+*   Automate [label](https://github.com/unifiedjs/github-tools/labels)/issue
+    interplay:
+    if maintainer adds `type/question`, the issue is closed and a predefined
+    response posted to ask in discussions instead;
+    `platform/*` can be found from the issue checklist;
+    merging can add `status/merge`, or `status/wontfix` can close one;
+    `area/*` could be added to the issue title?
+*   Maybe difficult, but: GH has support for finding similar issues, but doesn’t
```

Comment:
I often find these types of answers condescending, just because they're so rarely right.

Reply:
True!

Reply:
So I wouldn't do them, or I would note that when suggesting them.

Reply:
Removed it, thanks!